src/mjbots_robot_interface.cpp

```diff
@@ -96,10 +101,8 @@ MjbotsRobotInterface::MjbotsRobotInterface(const std::vector<JointMoteus> &joint
   moteus_data_.commands = {commands_.data(), commands_.size()};
   moteus_data_.replies = {replies_.data(), replies_.size()};
   moteus_data_.timeout = timeout_;
-
 }
 
-
```

Comment:
nit: white space

Reply:
Agreed, we should do a formatting run (see #35).

Reply:
Yea Ethan has been pushing for more auto formatting and I've been slowly coming around to the idea.